.github/problem-matcher-lint-cpplint.json

```diff
@@ -0,0 +1,17 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "lint-cpplint",
+            "severity": "error",
```

Comment:
Any reason for your severity inconsistencies with the matchers? I can't remember if it just defaults to error or something?

Edit, you've matched on the others! :facepalm:

Reply:
Error is the default, but I wanted to be explicit because my experience with the problem matchers has been that the documentation is...garbage unfortunately.

Reply:
Yeah makes perfect sense, I'd missed the regex matching elsewhere initially.